### PR TITLE
refactor: remove NULL; use nullptr instead - part 2

### PR DESCRIPTION
**Pull request by rodolforg**

### CODE REVIEW

**Comment by rodolforg** [comment]
Should I squash commits labeled as p2/p3/p4/p5 into the first commit or into a new one?

**Comment by ice0** [comment]
> Should I squash commits labeled as p2/p3/p4/p5 into the first commit or into a new one?

Better to squash into a new one. Or just rename them.

File: synfig-core/src/modules/mod_bmp/trgt_bmp.cpp [hunk 1]
```diff
@@ -150,10 +150,10 @@ bmp::bmp(const char *Filename, const synfig::TargetParam& params):
 	rowspan(),
 	imagecount(),
 	multi_image(false),
-	file(NULL),
+    file(nullptr),
```
Comment (ice0) [inline synfig-core/src/modules/mod_bmp/trgt_bmp.cpp:153]: tabs

File: synfig-core/src/modules/mod_dv/trgt_dv.cpp [hunk 2]
```diff
@@ -82,10 +82,10 @@ SYNFIG_TARGET_SET_VERSION(dv_trgt,"0.1");
 dv_trgt::dv_trgt(const char *Filename, const synfig::TargetParam & /* params */):
 	imagecount(0),
 	wide_aspect(false),
-	file(NULL),
+    file(nullptr),
```
Comment (ice0) [inline synfig-core/src/modules/mod_dv/trgt_dv.cpp:85]: tabs

File: synfig-core/src/modules/mod_imagemagick/trgt_imagemagick.cpp [hunk 3]
```diff
@@ -79,10 +79,10 @@ SYNFIG_TARGET_SET_VERSION(imagemagick_trgt,"0.1");
 imagemagick_trgt::imagemagick_trgt(const char *Filename,  const synfig::TargetParam &params):
 	imagecount(),
 	multi_image(false),
-	file(NULL),
```
Comment (ice0) [inline synfig-core/src/modules/mod_imagemagick/trgt_imagemagick.cpp:82]: tabs

File: synfig-core/src/modules/mod_magickpp/trgt_magickpp.h [hunk 4]
```diff
@@ -71,13 +71,13 @@ class magickpp_trgt : public synfig::Target_Scanline
 		width(),
 		height(),
 		filename(filename),
-		buffer1(NULL),
-		start_pointer(NULL),
-		buffer_pointer(NULL),
-		buffer2(NULL),
-		previous_buffer_pointer(NULL),
+	    buffer1(nullptr),
```
Comment (ice0) [inline synfig-core/src/modules/mod_magickpp/trgt_magickpp.h:74]: tabs

File: synfig-core/src/modules/mod_mng/trgt_mng.cpp [hunk 5]
```diff
@@ -97,18 +97,18 @@ mng_error_proc(mng_handle /*mng*/, mng_int32 /*error*/,
...
 	h(),
-	mng(NULL),
+    mng(nullptr),
 	multi_image(),
 	ready(false),
 	imagecount(),
 	filename(Filename),
-	buffer(NULL),
-	color_buffer(NULL),
+    buffer(nullptr),
```
Comment (ice0) [inline synfig-core/src/modules/mod_mng/trgt_mng.cpp:108]: tabs

File: synfig-core/src/modules/mod_noise/distort.cpp [hunk 6]
```diff
@@ -68,7 +68,7 @@ NoiseDistort::NoiseDistort():
 	Layer_CompositeFork(1.0,Color::BLEND_STRAIGHT),
 	param_displacement(ValueBase(Vector(0.25,0.25))),
 	param_size(ValueBase(Vector(1,1))),
-	param_random(ValueBase(int(time(NULL)))),
+    param_random(ValueBase(int(time(nullptr)))),
```
Comment (ice0) [inline synfig-core/src/modules/mod_noise/distort.cpp:71]: tabs

File: synfig-core/src/modules/mod_noise/noise.cpp [hunk 7]
```diff
@@ -68,7 +68,7 @@ SYNFIG_LAYER_SET_VERSION(Noise,"0.0");
 Noise::Noise():
 	Layer_Composite(1.0,Color::BLEND_COMPOSITE),
 	param_gradient(ValueBase(Gradient(Color::black(), Color::white()))),
-	param_random(ValueBase(int(time(NULL)))),
+    param_random(ValueBase(int(time(nullptr)))),
```
Comment (ice0) [inline synfig-core/src/modules/mod_noise/noise.cpp:71]: tabs

File: synfig-core/src/modules/mod_openexr/trgt_openexr.cpp [hunk 8]
```diff
@@ -62,9 +62,9 @@ exr_trgt::exr_trgt(const char *Filename, const synfig::TargetParam &params):
 	imagecount(0),
 	scanline(),
 	filename(Filename),
-	exr_file(NULL),
-	buffer(NULL),
-	buffer_color(NULL)
+    exr_file(nullptr),
```
Comment (ice0) [inline synfig-core/src/modules/mod_openexr/trgt_openexr.cpp:65]: tabs

File: synfig-core/src/modules/mod_png/trgt_png.cpp [hunk 9]
```diff
@@ -78,23 +78,23 @@ png_trgt::png_out_warning(png_struct *png_data,const char *msg)
 //Target *png_trgt::New(const char *filename){	return new png_trgt(filename);}
 
 png_trgt::png_trgt(const char *Filename, const synfig::TargetParam &params):
-	file(NULL),
-	png_ptr(NULL),
-	info_ptr(NULL),
+    file(nullptr),
```
Comment (ice0) [inline synfig-core/src/modules/mod_png/trgt_png.cpp:81]: tabs

File: synfig-core/src/modules/mod_ppm/trgt_ppm.cpp [hunk 10]
```diff
@@ -56,8 +56,8 @@ ppm::ppm(const char *Filename, const synfig::TargetParam &params):
 	multi_image(false),
 	file(),
 	filename(Filename),
-	color_buffer(NULL),
-	buffer(NULL),
+    color_buffer(nullptr),
```
Comment (ice0) [inline synfig-core/src/modules/mod_ppm/trgt_ppm.cpp:59]: tabs

**Comment by rodolforg** [comment]
Renamed the commits (and squashed p5+p4).
I believe I fixed all indentation issues.

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**